x-pack/plugins/security_solution/public/management/pages/policy/view/tabs/policy_tabs.tsx

```diff
@@ -7,15 +7,17 @@
 
```

Comment:
Note: Changes in this file might look drastic but in short:
* Turn the previous array of tabs into a key:value object (PolicyTab) with defined and typed keys (PolicyTabsKeys)
* Map the new object into an array to be used by the tabs component
* Add the logic to display or not and redirect the user based on the host isolation exceptions permissions.